sda-commons-server-auth-testing/README.md

```diff
@@ -11,11 +11,12 @@ testCompile 'org.sdase.commons:sda-commons-server-auth-testing:<current-version>
 ```
 
 In an integration test, authentication can be configured using the 
-[`AuthRule`](./src/main/java/org/sdase/commons/server/auth/testing/AuthRule.java) and/or the [`OpaRule`](./src/main/java/org/sdase/commons/server/opa/testing/OpaRule.java).
+[`AuthRule`](./src/main/java/org/sdase/commons/server/auth/testing/AuthRule.java) and/or the [`OpaRule`](./src/main/java/org/sdase/commons/server/opa/testing/OpaRule.java) (Junit 4) or
+[`AuthExtension`](./src/main/java/org/sdase/commons/server/auth/testing/AuthExtension.java) and/or the [`OpaExtension`](./src/main/java/org/sdase/commons/server/opa/testing/OpaExtension.java) (Junit 5).
 
-## Auth Rule
-The `AuthRule` uses the `EnvironmentRule` to create the `AuthConfig` in an environment property called `AUTH_RULE`.
-Therefore the configuration in the test needs to use this property and the application is required to use the 
+## Auth Extension
```

Comment:
Please don't replace the documentation of JUnit 4 features but add documentation for JUnit 5 extensions.